pkg/resources/failover_group_acceptance_test.go

```diff
@@ -181,6 +181,18 @@ func TestAcc_FailoverGroupInterval(t *testing.T) {
 					resource.TestCheckResourceAttr("snowflake_failover_group.fg", "replication_schedule.0.interval", "10"),
 				),
 			},
+			// Remove replication schedule
```

Comment:
can we not make it the last step before the import? (separate test or different ordering) We are now testing import without replication schedule

Reply:
👍 Reordered